main.py

```diff
@@ -1,42 +1,128 @@
+id_user = 0
+balance = 100
+state = None
+list_of_users = []
 
-class Atm:
 
-    @staticmethod
-    def reg():
-        global newLogin
-        global newPassword
-        print('Create your login: ')
-        newLogin = input()
-        print('Create your password: ')
-        newPassword = input()
+def reg():
+    global id_user
+    print('Create your login: ')
+    new_login = input()
+    print('Create your password: ')
+    new_password = input()
+    for l in range(len(list_of_users)):
```

Comment:
it's better to call variables understandable names. 
And it's also a good practice to not call variables `l` because it looks like 1